CMakeLists.txt

```diff
@@ -206,10 +206,14 @@ install(
   RUNTIME DESTINATION bin
   INCLUDES
   DESTINATION include)
+  
+# Install the header files so other ROS packages can find them
```

Comment:
`# Install the header files so other ROS packages can find them` should be:
`# Install the header files so other packages depending on this one can find them`